Source/Blazorise/Styles/_bar.scss

```diff
@@ -0,0 +1,405 @@
+    top: 0;
+    padding: 0;
+    min-width: var(--b-sidebar-width, 220px);
+    max-width: var(--b-sidebar-width, 220px);
+    width: var(--b-sidebar-width, 220px);
+    transition: width 200ms ease-in-out, min-width 200ms ease-in-out;
+    box-shadow: 2px 0 6px rgba(0,21,41,.35);
+    height: 100vh;
+
+    .b-bar-menu {
+        height: calc(100% - var(--b-sidebar-brand-height, 64px));
+        justify-content: space-between;
+        flex-direction: column;
+        display: flex !important;
+    }
+
+    .b-bar-brand {
+        width: 100%;
+        display: flex;
+        margin: 0;
+        height: var(--b-sidebar-brand-height, 64px);
+    }
+
+    .b-bar-item {
+        margin: auto;
+        flex-grow: 1;
+        min-height: 40px;
+
+        .b-bar-icon {
+            font-size: 18px;
+            margin: 3px;
+        }
+    }
+
+    .b-bar-start {
+        width: 100%;
+        display: inline-block;
+    }
+
+    .b-bar-end {
+        padding-bottom: 1rem;
+        width: 100%;
+        padding-top: 1rem;
+        display: inline-block;
+    }
+
+    .b-bar-link {
+        display: block;
+        width: 100%;
+        text-decoration: none;
+        padding: .5rem .5rem .5rem 1.5rem;
+        cursor: pointer;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow-x: hidden;
+    }
+
+    .b-bar-label {
+        background: transparent;
+        color: #adb5bd;
+        padding: .375rem 1.25rem;
+        font-size: .75rem;
+        text-overflow: ellipsis;
+        overflow-x: hidden;
+    }
+
+    .b-bar-dropdown {
+        &:before {
+            @include b-bar-dropdown-arrow;
+            -webkit-transform: rotate(-135deg);
+            transform: rotate(-135deg);
+            top: 1.5rem;
+        }
+
+        &:not(.show):before {
+            -webkit-transform: rotate(45deg);
+            transform: rotate(45deg);
+            top: 1.3rem;
+        }
+
+        .b-bar-dropdown-menu {
+            display: none;
+            background: inherit;
+            color: inherit;
+            float: none;
+            padding: 5px 0px;
+
+            &.show {
+                display: block;
+            }
+
+            .b-bar-dropdown-item {
+                position: relative;
+                color: inherit;
+                transition: background 100ms ease-in-out,color 100ms ease-in-out;
+                text-decoration: none;
+                display: block;
+                width: 100%;
+                white-space: nowrap;
+                text-overflow: ellipsis;
+                overflow-x: hidden;
+
+                i {
+                    margin-right: 0.3rem;
+                }
+            }
+
+            &:before {
+                background: inherit;
+                box-shadow: none;
+            }
+        }
+    }
+
+    .b-bar-mobile-toggle {
+        right: 20px;
+        margin: auto;
+        display: none;
+    }
+    /* 
+        Theming
+    */
+    &.b-bar-dark {
```

Comment:
@stsrki these are the two sections I was referring to that I think should be generated in the `ThemeGenerator`.
` &.b-bar-dark` and `&.b-bar-light` + their children.

As well as `:root` at the top of this file